runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -55,206 +54,186 @@
 public class CoordinateClue extends ClueScroll implements LocationClueScroll
 {
 	@Getter
+	@Builder
```

Comment:
At this point, CoordinateClue itself should use a builder pattern and incorporate the fields from CoordinateClueInfo.